Examples/Scripts/SetUpHCPPipeline-long.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/echo This script should be sourced before calling a pipeline script, and should not be run directly:
+
+#Don't edit this line
+SAVEHCPPIPE="${HCPPIPEDIR:-}"
+
+## Edit this line: environment variable for location of HCP Pipeline repository
+## If you leave it blank, and $HCPPIPEDIR already exists in the environment,
+## that will be used instead (via the SAVEHCPPIPE variable, defined above)
+#export HCPPIPEDIR=/home/mmilchenko/src/HCPpipelines-4.7.0
+#export HCPPIPEDIR=/home/mmilchenko/src/HCPpipelines-longitudinal_fs
+export HCPPIPEDIR=/media/myelin/brainmappers/Connectome_Project/MishaLongitudinal/src/HCPpipelines-longitudinal_fs
```

Comment:
I don't think we want any of our internal settings committed.  I am guessing you didn't need any new exports for longitudinal, so we shouldn't have an additional SetUp file for it.

Reply:
Agree

Reply:
ok